.github/workflows/eslint.yaml

```diff
@@ -0,0 +1,18 @@
+name: JavaScript Lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install npm packages
+        run: npm install eslint@latest --save-dev
+      - name: Install eslint React Plugin
+        run: npm install eslint-plugin-react@latest --save-dev
+      - name: Install Google style plugin
+        run: npm install eslint-config-google@latest --save-dev
+      - name: Lint with ESLint
+        run: ./node_modules/.bin/eslint frontend/**/*.js
```

Comment:
This might be a nit but can you explain what this file does ?

Reply:
Sure! So in order to successfully run the linter, there are some preconditions which need to be satisfied (i.e plugins, packages, and certain configurations that need to be installed). This file makes sure that the correct plugins and packages are installed for the .eslintrc.json file to correctly run and actually runs the necessary commands to lint. Side note: If you look at the details in the JS lint check on this PR, you can see these steps in the log.

Reply:
This yaml file, like other linter/formatter, is for us to configure a Ubuntu machine to tell Github to run a "workflow" during the "[pull_request]". All such files should go to .github/workflows(https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow). For this specific file, we want to configure a workflow to run eslint, which is a JavaScript linter, during pull request. Like Dhruvi said, the steps before the eslint is to make sure the dependencies are installed on that machine Github provides us.